app/drivers/google/driver.go

```diff
@@ -394,21 +459,52 @@ func (p *config) DestroyInstanceAndStorage(ctx context.Context, instances []*typ
 			}
 		}
 
-		requestID := uuid.New().String()
-
-		_, err = p.deleteInstance(ctx, p.projectID, zone, instance.ID, requestID)
-		if err != nil {
+		instanceDeleteOperation, deleteInstanceErr := p.deleteInstance(ctx, p.projectID, zone, instance.ID, uuid.New().String())
+		if deleteInstanceErr != nil {
 			// https://github.com/googleapis/google-api-go-client/blob/master/googleapi/googleapi.go#L135
-			if gerr, ok := err.(*googleapi.Error); ok &&
+			if gerr, ok := deleteInstanceErr.(*googleapi.Error); ok &&
```

Comment:
this would be a behaviour change, since earlier we were returning `err` and now changed the variable name to `deleteInstanceErr`.
You can assign err = deleteInstanceErr after `if deleteInstanceErr != nil` check

Reply:
Fixed